otx/algorithms/detection/configs/detection/resnet50_deformable-detr/template_experimental.yaml

```diff
@@ -0,0 +1,64 @@
+# Description.
+model_template_id: Custom_Object_Detection_Gen3_Deformable_DETR
+name: Deformable_DETR
+task_type: DETECTION
+task_family: VISION
+instantiation: "CLASS"
+summary: Class-Incremental Object Detection for Deformable_DETR
+application: ~
+
+# Algo backend.
+framework: OTXDetection v2.9.1
+
+# Task implementations.
+entrypoints:
+  base: otx.algorithms.detection.adapters.mmdet.task.MMDetectionTask
+  openvino: otx.algorithms.detection.adapters.openvino.task.OpenVINODetectionTask
+  nncf: otx.algorithms.detection.adapters.mmdet.nncf.task.DetectionNNCFTask
+
+# Capabilities.
+capabilities:
+  - compute_representations
+
+# Hyperparameters.
+hyper_parameters:
+  base_path: ../configuration.yaml
+  parameter_overrides:
+    learning_parameters:
+      batch_size:
+        default_value: 2
+        auto_hpo_state: POSSIBLE
+      learning_rate:
+        default_value: 0.0002
+        auto_hpo_state: POSSIBLE
+      learning_rate_warmup_iters:
+        default_value: 3
+      num_iters:
+        default_value: 12
```

Comment:
It's just a question. Is there any reason why it's different from models?

Reply:
If you ask why deformable detr is different with other detectors(ATSS, SSD), the answer is current deformable detr does not support ReducedLRPlateau decay. The LR scheduler don't works properly. I'll modify this in another PR.

If you ask why template.yaml is different from model.py, then the answer also same. It will be modified in another PR.

Reply:
Sorry for my ambiguous. I just want to ask about `num_iters` default value.

Reply:
Current Deformable DETR does not support earlystopping, same reason with ReducedLRPlateau. I'll modify this in another PR.